test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java

```diff
@@ -396,28 +406,54 @@ public void TestQueryBatch(String inputFileName) throws SQLException, ClassNotFo
         BufferedWriter bw = new BufferedWriter(fw);
         batch_run.batch_run_sql(connection_bbl, bw, testFilePath, logger);
         bw.close();
-        
+        if(sla == 0){
+            sla = defaultSLA*1000000L;
+        }
+        int n = exec_times.size();
+        int index = n - num_lines;
+        long total_time = 0;
+        for(int i = index; i < n; i++){
+            total_time += exec_times.get(i);
```

Comment:
Can't we just sum all the values in exec_times blindly. I didn't quite understand the index logic here